app/assets/javascripts/application.js

```diff
@@ -544,6 +544,43 @@ function closeModalSpinner(spinnerTarget, modalTarget, callback) {
     $(modalTarget).modal('hide');
 }
 
+// handles showing/hiding main message_modal and cleaning up state on full & partial page renders
+function showMessageModal(notice=null, alert=null) {
+    console.log('calling showMessageModal');
```

Comment:
[`console.log` in production](https://www.google.com/search?q=console.log+in+production) is generally discouraged.  It's fine in development, but in production omitting such debugging code is good form.  Breakpoints in DevTools are a cleaner and more robust solution if we need to debug in production.

```suggestion
```

Reply:
Yeah, that's a debug statement I forgot to take out.  Will remove.